### DISCUSSION TITLE
How to structure my application inside app folder?

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [vercel/next.js#43826] [Help]
Hi. It is so confusing to me when using app directory. I don't know if I should put just my routes specific files that mentioned in the next 13 docs or I can put any other folder that I use to structure my application inside `app` folder including `components` , `database` related modules or any other folder that I use to structure my code. If `app` folder is just for routes related files then why next team named it `app` and not `routes` or any  name that suitable for this purpose? or if  I can put any folder inside `app` folder then please help me understand how to structure my application inside `app` folder that does not interfere with my routes folders? Thanks.

**Comment by ciruz** [comment]
Hey for example like so => https://github.com/vercel/app-playground
